drivers/xen/events.c

```diff
@@ -42,7 +116,6 @@ int bind_event_channel(evtchn_port_t port, evtchn_cb_t cb, void *data)
 	__ASSERT(cb != NULL, "%s: NULL callback for evtchn #%u\n",
 		__func__, port);
 
-
```

Comment:
This is just code style fix, 1 empty line instead of 2.

Reply:
Ok. I am not sure what's Zephyr policy here. For Xen, we would tend to avoid doing clean-up in the same commit (or it is at least mentioned in the commit message).

This is zephyr, so please follow their process :).